scalafix/README.md

```diff
@@ -1,22 +1,35 @@
 # Scalafix rules for cats
 
-## Try this!
+## How to use
 
-[Install the Scalafix sbt plugin](https://scalacenter.github.io/scalafix/docs/users/installation)
+1. [Install the Scalafix sbt plugin](https://scalacenter.github.io/scalafix/docs/users/installation)
 
-To run all rules that apply to version `1.0.0-RC1` run
+2. Configure the SemanticDB compiler plugin to enable synthetics:
```

Comment:
How about moving this to the "Migration to Cats v2.2.0" section since it is only required for this rule and not Cats_v1_0_0? Future rules also might require different `scalacOptions` or no additional options at all.

Reply:
done